src/status_im/commands/core.cljs

```diff
@@ -23,20 +24,26 @@
   {:events [::accept-request-address-for-transaction]}
   [{:keys [db]} message-id address]
   {:db (dissoc db :commands/select-account)
-   ::json-rpc/call [{:method "shhext_acceptRequestAddressForTransaction"
+   ::json-rpc/call [{:method (if config/waku-enabled?
+                               "wakuext_acceptRequestAddressForTransaction"
+                               "shhext_acceptRequestAddressForTransaction")
```

Comment:
I'm no Clojure wizard but it seems to me like all of thse `if config/waku-enabled?` could be replaced with something like:
```clojure
get-rpc-call-prefix + "acceptRequestAddressForTransaction"
```
Not sure how concatenation works in Clojure.
or
```clojure
get-rpc-call "acceptRequestAddressForTransaction"
```